src/controllers/auth-controller.js

```diff
@@ -26,7 +26,7 @@ async function login (req, res, next) {
       throw new NotFoundError(userNotFoundError);
     }
 
-    const token = await TokenLib.createUserToken({ _id: user._id, email, role: user.role });
+    const token = await TokenLib.createUserToken({ userId: user._id, email, role: user.role });
```

Comment:
when changing such logic, keep in mind it will affect all places, where req.userData._id is used and you need to change them all